src/SprykerEco/Yves/Computop/Mapper/Init/PostPlace/AbstractPostPlaceMapper.php

```diff
@@ -28,6 +28,7 @@ public function createComputopPaymentTransfer(TransferInterface $quoteTransfer)
         $computopPaymentTransfer->setCurrency($this->store->getCurrencyIsoCode());
         $computopPaymentTransfer->setResponse(ComputopConfig::RESPONSE_ENCRYPT_TYPE);
         $computopPaymentTransfer->setClientIp($this->getClientIp());
+        $computopPaymentTransfer->setReqId($this->generateReqId($quoteTransfer));
```

Comment:
Isn't the value is already there after PrePlace calls ?
Also I see a lot similar things in `AbstractPrePlaceMapper` and `AbstractPostPlaceMapper`
I would think of extracting those to a model or at least moving to a single abstract `XxxxMapper` class